opm/simulators/wells/WellStateFullyImplicitBlackoil.hpp

```diff
@@ -337,14 +337,14 @@ namespace Opm
         }
 
         /// One current control per group.
-        void setCurrentInjectionGroupControl(const std::string& groupName, const Group::InjectionCMode& groupControl ) {
-            current_injection_group_controls_[groupName] = groupControl;
+        void setCurrentInjectionGroupControl(const Opm::Phase& phase, const std::string& groupName, const Group::InjectionCMode& groupControl ) {
```

Comment:
I would think it more natural to have `groupName` first, then `phase`, since the original call has only `groupName`. Is there a specific reason for preferring the current order?

Reply:
It is the logic used in opm-common. I think we should keep the same order as in opm-common